docs/releases/1.9.txt

```diff
@@ -416,6 +416,15 @@ Miscellaneous
   Therefore, the ``@skipIfCustomUser`` decorator is no longer needed to
   decorate tests in ``django.contrib.auth``.
 
+* In Generic Class Based Views,
+  :meth:`~django.views.generic.edit.FormMixin.get_context_data` was added to
+  :class:`~django.views.generic.edit.FormMixin` and form initialization was moved from
+  :class:`~django.views.generic.edit.ProcessFormView`'s :meth:`~django.views.generic.edit.ProcessFormView.get`
+  method to the new method. This may be backwards incompatible if you have
+  overridden the ``get_context_data`` method without using ``super()`` function
```

Comment:
I'm not sure about this, you end up with: 'This may be backwards incompatible if you have overridden the
  ``get_context_data()`` method without calling ``super()`` in an overridden
  ``get_context_data()`` method.'

.. which doesn't sound right. I think my version sounds clunky too. I think there's two good options here:

1. If user understands how to use super(), considering that all super() calls are similar, there's no need for further explanation or example, it's enough to say: 'This may be backwards incompatible if you have
+  overridden the ``get_context_data`` method without using ``super()`` call.' 

2. If the user does NOT understand super() calls, it's hard to explain what needs to be done and why, it would be easier for the user to look at super() calls in other methods in that module or we could give an example in the doc which would be more succinct and clearer than trying to explain it.